gui-libs/tepl/tepl-6.2.0.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org meson virtualx
+
+DESCRIPTION="GtkSourceView-based text editors and IDE helper library"
+HOMEPAGE="https://gitlab.gnome.org/swilmet/tepl"
+
+LICENSE="LGPL-3+"
+SLOT="6"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~riscv ~sparc ~x86"
+IUSE="+introspection gtk-doc"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	>=dev-libs/glib-2.62:2
+	>=x11-libs/gtk+-3.22:3[introspection?]
+	>=x11-libs/gtksourceview-4.0:4[introspection?]
+	>=gui-libs/amtk-5.0:5[introspection?]
+	dev-libs/icu:=
+	introspection? ( >=dev-libs/gobject-introspection-1.42:= )
+"
+RDEPEND="${DEPEND}
+	gnome-base/gsettings-desktop-schemas
+"
```

Comment:
The ebuild diff vs the previous version is

````diff
-DEPEND="${RDEPEND}"
+RDEPEND="${DEPEND}
+	gnome-base/gsettings-desktop-schemas
+"
````

and the `pkgdiff -b` diff is

````diff
 tepl_private_deps = [
   dependency('icu-uc'),
   dependency('icu-i18n'),
+  dependency('gsettings-desktop-schemas')
 ]
````

So this looks wrong for a couple reasons—`RDEPEND` is now defined twice and it uses the undefined `DEPEND`, and now `DEPEND` isn't defined. But moreover, shouldn't `gnome-base/gsettings-desktop-schemas` just go right after `icu` in the dependency list?